stripe/src/test/java/com/stripe/android/model/SourceParamsTest.java

```diff
@@ -1,6 +1,7 @@
 package com.stripe.android.model;
 
 import android.support.annotation.NonNull;
+import android.util.Log;
```

Comment:
Nit: I think this is probably a hold over?

FYI - Log is for Android logcat when you're running code on an Android device. You'll want system.out.print if you want to see logs appear from the tests. 
Android Studio also has an optimize imports setting that will do it on the fly for you:
"Settings->Editor->Auto Import->Optimize Imports on the fly"

Reply:
Ah nice, yeah it was leftover while I was debugging a test failure